### DISCUSSION TITLE
Save option not working after first time

### DISCUSSION THREAD

**Discussion by tsandun** [burhanrashid52/PhotoEditor#420] [General]
api 'ja.burhanrashid52:photoeditor:1.1.0' working android device gracefully but in Huawei device sometime not saving.In the app install first time its working good.But after first time nothing happen.Progress didint hide.After debugging in library file i got an error from asycntask => "An error occurred while executing doInBackground() ".

**Comment by burhanrashid52** [comment]
Can you share the stack trace of the error ?

**Comment by tsandun** [comment]
There is no exception from my code.I found the issue. It comes from
Asynctask.java File. It gets a run time exception.
[image: image.png]
